specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/examples/communityGalleryExamples/CommunityGallery_Get.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "location": "myLocation",
+    "api-version": "2022-08-03",
+    "publicGalleryName": "publicGalleryName"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "location": "myLocation",
+        "name": "publicGalleryName",
+        "type": "Microsoft.Compute/Locations/CommunityGallery",
+        "identifier": {
+          "uniqueId": "/CommunityGalleries/publicGalleryName"
+        },
```

Comment:
This part of the payload is very confusing to me. Why isn't the resource identifier already a unique id? Why isn't there an 'id' proeprty  in the response, for that matter???

Reply:
looks like its been that way for a while so there are probably some barriers to fixing it :(

Reply:
This is proxy only resource and they are not bounded to any subscription/resourceGroup, so the typical id does not work in this case and we introduced 'identifier-uniqueId' for this purpose.